src/aideck/embeddings/fastembed_provider.py

```
@@ -0,0 +1,462 @@
+                    embeddings_iterator = self.embedding_model.query_embed(texts)
+                elif self.config.doc_embed_type == "passage":
+                    embeddings_iterator = self.embedding_model.passage_embed(texts)
+                else:
+                    embeddings_iterator = self.embedding_model.embed(texts)
+                
+                embeddings = self._process_embeddings(embeddings_iterator)
+            
+            if self.config.normalize_embeddings:
+                embeddings = self._normalize_embeddings(embeddings)
+            
+            return embeddings
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)